realm/realm-library/src/androidTestObjectServer/java/io/realm/util/SyncTestUtils.java

```diff
@@ -116,4 +129,14 @@ public static AuthenticateResponse createRefreshResponse() {
     public static AuthenticateResponse createErrorResponse(ErrorCode code) {
         return AuthenticateResponse.from(new ObjectServerError(code, "dummy"));
     }
+
+    public static void resetSyncMetadata() {
+        try {
+            SYNC_MANAGER_RESET_METHOD.invoke(null);
+        } catch (InvocationTargetException e) {
```

Comment:
it requires API 19 unfortunately 
<img width="1235" alt="screen shot 2017-06-02 at 12 51 03" src="https://cloud.githubusercontent.com/assets/1793238/26724617/5e50983c-4792-11e7-8fc7-33d7b6f7c239.png">